src/lib/routes/admin-api/project/environments.ts

```diff
@@ -60,7 +69,13 @@ export default class EnvironmentsController extends Controller {
                 openApiService.validPath({
                     tags: ['Projects'],
                     operationId: 'removeEnvironmentFromProject',
-                    responses: { 200: emptyResponse },
+                    summary: 'Remove a environment to a project.',
+                    description:
+                        'This endpoint removes the provided environment to the specified project, with optional support for enabling and disabling change requests for the environment and project.',
+                    responses: {
+                        200: emptyResponse,
+                        ...getStandardResponses(400, 401, 403),
```

Comment:
Returns a 400 when trying to delete an environment from a junk project

Reply:
That's a good question! I'll add a task for this, maybe we should discuss that there? Seems like there should be some kind of decision/guideline on this kind of use case, doubt this would be the only instance of it?